### ISSUE TITLE
ReferenceError: AbortController is not defined in Chrome 65 (and earlier)

### ISSUE THREAD

**Issue by aleksejleonov** [maplibre/maplibre-gl-js#783]
**maplibre-gl-js version**: 2.0.0

**browser**: Chrome 65 (or earlier versions)

### Steps to Trigger Behavior

 1. Open https://maplibre.org/maplibre-gl-js-docs/example/simple-map/ in Chrome 65 (or earlier versions).

### Link to Demonstration

Tested with Browserstack.

**Chrome 65** (`ReferenceError: AbortController is not defined`)
<img width="1082" alt="Screenshot 2022-01-15 at 18 09 09" src="https://user-images.githubusercontent.com/241582/149629381-7256e695-9ccc-4d2c-bee4-4724091c254d.png">
<img width="1159" alt="Screenshot 2022-01-15 at 18 09 32" src="https://user-images.githubusercontent.com/241582/149629386-ae9624c0-8e2f-4819-a199-fce7e66e4a9d.png">

**Chrome 66** (OK)
<img width="1079" alt="Screenshot 2022-01-15 at 18 10 19" src="https://user-images.githubusercontent.com/241582/149629393-19e4f5de-799d-4355-896a-93a897a27a87.png">

### Expected Behavior

I don't believe it makes any sense to support old Chrome 49–65 versions :)
Chrome 66 was released ~4 years ago and v2 works fine in it.

But it would be useful to specify Chrome 66 as the minimum version in the Release notes as breaking change.

### Actual Behavior

Map breaks in Chrome 65 (or earlier versions).

**Comment by HarelM** [comment]
Feel free to submit a PR although I don't a real reason to mention this, as you said, a version from 4 years ago...

**Comment by aleksejleonov** [comment]
@HarelM , please check #784.

**Comment by HarelM** [comment]
Resolved by #784

**Comment by acalcutt** [comment]
Just an FYI for anyone interested, not to say anyone should do this.

I fixed this error in my [IE/ES5 release](https://github.com/acalcutt/maplibre-gl-js/tree/internetexplorer_test3) by adding the following to [src/util/ajax.ts](https://github.com/acalcutt/maplibre-gl-js/commit/6317485e0eaf9b6bee6a32373d53a349d0a6f0b0)

```
import 'promise-polyfill/src/polyfill';
import 'unfetch/polyfill';
import 'abortcontroller-polyfill';
```

This basically adds support for fetch and abortcontroller in browsers that don't support it. I think support for fetch and abortcontroller used to be added by the older version of buble, but buble is no longer used in the build process. 

In the IE/ES5 version I mention above I added buble back, but the newer versions no longer add this fetch/abortcontroller backward compatibility. I used the code above in src/util/ajax.ts to fix that in that release. you can see that version still supports the older browsers with this example page ( https://wifidb.net/demo/abortcontroller/test.html ) .
